tests/lib/rules/no-reserved-component-names.js

```diff
@@ -374,8 +364,6 @@ const invalidElements = [
   'Blink',
   'center',
   'Center',
-  'command',
-  'Command',
```

Comment:
Where did the list previously come from? Is it okay that the `content`, `element`, `shadow`, `discard`, `unknown` and `command` names are now allowed?

Reply:
It seems they were listed on MDN.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element#obsolete_and_deprecated_elements

https://github.com/vuejs/eslint-plugin-vue/pull/757#issuecomment-452968477

But they have been updated and some are not currently listed.

We don't know where the old ones came from, so I think it's hard to keep track of them. So I think it makes sense to use the typescript definition.

Reply:
`content` and `shadow` are still listed there. Maybe it makes sense to have an additional list next to the one pulled from the TypeScript definition?

Or should we strive to get those elements included in TypeScript itself?

Reply:
> Apparently, the TypeScript types are generated from https://github.com/mdn/browser-compat-data.

Thank you for letting me know. I didn't know that.
Hmm. It might make more sense to generate resource files from `@mdn/browser-compat-data` than from `typescript`. What do you think?

https://www.npmjs.com/package/@mdn/browser-compat-data

Reply:
Yeah, skipping the intermediate TypeScript step seems to make sense.

Reply:
I'll look into how to do that.

Reply:
Hmm, so BCD+WhatWG include everything that TypeScript has, plus `portal` HTML element and `discard` SVG element. Sounds like this is the better data source then.

However, running inside the version script might add new restricted elements (which could be a breaking change) only right before a new release. That seems dangerous to me.

Maybe we can try to make the script faster by not reconstructing the whole DOM with `jsdom`, but rather using a faster parser like [`node-html-parser`](https://www.npmjs.com/package/node-html-parser)?

Reply:
> However, running inside the version script might add new restricted elements (which could be a breaking change) only right before a new release. That seems dangerous to me.

So, I will to separate it from the update script and change it to check periodically with GitHub Actions.

> Maybe we can try to make the script faster by not reconstructing the whole DOM with jsdom, but rather using a faster parser like [node-html-parser](https://www.npmjs.com/package/node-html-parser)?

Thanks for sharing. I will check that out.

Reply:
I tried node-html-parser but it doesn't seem to work.
I suspect that the reason is that HTML with omitted `</html>` and `</body>` cannot be parsed well 🤔

Reply:
FYI: The `discard` SVG element seems to have been removed from the latest data.
https://github.com/mdn/browser-compat-data/pull/19782